packages/pictogrammers-1.0.0.yaml

```diff
@@ -0,0 +1,5 @@
+name: pictogrammers_icons
+description: Pictogrammers Icons, based on Material Design
```

Comment:
The description should be from the package.
If it's not specified in the package.yaml file, then it should be the first paragraph of the Readme.

I'm fine either way:
- adding this description line to the README, or
- making this description line longer by taking the whole paragraph from the README.